set.mm

```diff
@@ -491249,7 +491249,7 @@ of the indicator function (Contributed by Thierry Arnoux,
     hgt750lemg.n $e |- ( ph -> N : ( 0 ..^ 3 ) --> NN ) $.
     hgt750lemg.l $e |- ( ph -> L : NN --> RR ) $.
     hgt750lemg.1 $e |- ( ph -> N e. R ) $.
-    $( Lemma for the statement 7.50 of [Helfgott] p. 69.  : Applying a
+    $( Lemma for the statement 7.50 of [Helfgott] p. 69. : Applying a
        permutation ` T ` to a the 3 factors of a product does not change the
```

Comment:
Do the markup rules allow you to write "p. 69: applying" ? If not, simply remove the semicolon.
The permutation seems to be called "N" or "F" (I can't read the whole statement), but not "T"; typo: "a the"; I would write "three" instead of "3".

Reply:
I'm not sure whether your suggestion is allowed, so I've simply removed the semicolon, and fixed the typo, as well as used "three".

` T ` is actually really the permutation here (you can see it if you check the full change set).

Reply:
Ok, now I see. So F is the pullback by T and (F(N))(2) = N(T(2)) for instance.  There's probably a reason why you wrote it that way, I'll have a look at the website pages when updated.